.circleci/config.yml

```diff
@@ -79,6 +79,27 @@ jobs:
           name: Run integration tests
           command: cp workspace/*.linux_amd64 ./ && make test-integration
 
+  release:
+    <<: *docker_build_image
+    environment:
+      GOPATH: /go
+    steps:
+      - checkout
+      - run:
+          name: Release
+          command: |
+            if git tag | grep "v$(cat DRAUPNIR_VERSION)"; then
```

Comment:
I'd prefer to be explicit with `git tag -l "v$(cat DRAUPNIR_VERSION)" = "v$(cat DRAUPNIR_VERSION)".

`git tag` is an implicit `git tag -l` and then we're doing grep for no good reason IMO